README.md

```diff
@@ -4,6 +4,14 @@
 
 ## Development
 
+### Dependencies:
+
+To install poetry and pre-commit:
+```bash
+python3 -m pip install --user poetry
+python3 -m pip install pre-commit
```

Comment:
You do not need to install pre-commit separately. Poetry handles that for us.

Reply:
ok